fs/fs_zenfs.h

```diff
@@ -6,7 +6,14 @@
 
 #pragma once
 
+#if __cplusplus < 201703L
+#include <boost/filesystem.hpp>
```

Comment:
Maybe we should also check whether boost exists, if not, we can provide another solution for developers.....

Reply:
Lets make sure that we find a solution that also works for TerakDB. Not sure they want boost. Maybe we can detect building under TerakDB and then include TerakDB path logic somehow.

Reply:
I will provide a draft of an implementation that imports the and namespaces the `Path` class from a different file (which can eventually live in a TerakDB).

Reply:
@royguo I am considering to namespace the `experimental/filesystem` library for c++11. https://en.cppreference.com/w/cpp/header/experimental/filesystem
Would that be something that works out for you?
Also what compiler are you using?

Reply:
We can't use experimental/filesystem because it does not come with the `lexically_nomal` function. So we are back to the `Path` class in a different file.